package-configuration/linux-cli/test_cli_install.py

```diff
@@ -31,4 +37,16 @@ def setUp(self):
         self.get_cloudify = get_cloudify
 
     def test_full_cli_install(self):
-        self.run_get_cloudify('-f -v -e=/tmp/temp_env/ --nosudo')
+        tempdir = tempfile.mkdtemp()
+        self.run_get_cloudify('-f -v -e={0} --nosudo'.format(tempdir))
+        p = self.get_cloudify.run('{0}/bin/cfy --version'.format(tempdir))
```

Comment:
os.path.join is not enough here to support windows. we also need to support \\scripts\\.
Let's deal with that once we write tests for Windows.